src/main/java/com/github/s4u/plugins/PGPKeysCache.java

```diff
@@ -55,6 +55,9 @@ public PGPKeysCache(Log log, File cachePath, String keyServer) throws URISyntaxE
         if ("hkp".equalsIgnoreCase(uri.getScheme())) {
             scheme = "http";
             port = 11371;
+        } else if ("hkps".equalsIgnoreCase(uri.getScheme())) {
+            scheme = "https";
+            port = 443;
```

Comment:
For beginning is ok. I see here some of factory or strategy design for support multiple kind of connection.